src/main/java/Control/Scenario/UC_Auth.java

```diff
@@ -58,6 +60,11 @@ public UC_Auth(@NotNull DbProvider dbProvider) {
         try {
             db.beforeSqlExecution(true);
 
+            if (EmailValidator.getInstance().isValid(auth.getUser().getEmail()) == false) {
+                jsonResponse.setMessage("Presented email is not valid for use.");
+                throw new UserCreationException("Presented email is not valid for use.");
+            }
+
```

Comment:
ja som za taketo riesenie, super ze si na to rovno nasiel kniznicu a nemusel si si to sam pisat

Reply:
Ani by som to sam nepisal :D